### PR TITLE
💄 Add background-image

### LINKED ISSUE #58: Add background image and style main layout
In the desktop window, we like to add a full screen background image. The existing layers should be transparent.

### PR DESCRIPTION
**Pull request by monavorschulze**

### CODE REVIEW

File: src/app/App.tsx [hunk 1]
```diff
@@ -9,27 +9,30 @@ import Ads from './components/Ads/Ads';
...
-          </>
-        ) : (
-          <Feed onMatchClick={setTargetMatchId} />
-        )}
-      </main>
-      <aside className={classes.aside}>
-        <User />
-        <Ads />
-      </aside>
+    <div>
```
Comment (lmachens) [inline src/app/App.tsx]: You added a `div` here. I don't think it's required :).

File: src/app/globals.css [hunk 2]
```diff
@@ -9,10 +9,10 @@
...
   --color-secondary: #e2e2e2;
   --color-shade-first: #060606;
-  --color-shade-second: #1b1f24;
-  --color-shade-third: #23272c;
-  --color-shade-fourth: #393d43;
-  --color-shade-fifth: #585d64;
+  --color-shade-second: rgb(27, 31, 36, 0.4);
+  --color-shade-third: rgb(27, 31, 36, 0.4);
+  --color-shade-fourth: rgb(27, 31, 36, 0.4);
+  --color-shade-fifth: rgb(27, 31, 36, 0.4);
```
Comment (lmachens) [inline src/app/globals.css]: Is it intended that these colors have an alpha channel? The background-image will be slightly visible.

File: src/app/globals.css [hunk 3]
```diff
@@ -34,6 +34,7 @@ body {
   font-family: var(--font-family-primary);
   background: var(--color-shade-first);
   color: var(--color-secondary);
+  background-image: url('backgroundimage.jpg');
```
Comment (lmachens) [inline src/app/globals.css]: I would rename the file to `background.jpg`. All `.jpg` files are `images`.

**Review by lmachens — CHANGES_REQUESTED** [review]
Nice image

File: src/app/App.tsx (hunk 1 above)
Comment (monavorschulze) [reply src/app/App.tsx]: You're right :)

File: src/app/globals.css (hunk 3 above)
Comment (monavorschulze) [reply src/app/globals.css]: okay. done 👍

File: src/app/globals.css (hunk 2 above)
Comment (monavorschulze) [reply src/app/globals.css]: Some components have a slightly opacity in the layout. I woud like to updatet the css-globals colors later, when everyone have finished the components.

File: src/app/App.tsx (hunk 1 above)
Comment (AlexisRoe) [reply src/app/App.tsx:12]: But didn`t we need the outer div container to arrange everything inside with the grid?

**Review by AlexisRoe — COMMENTED** [review]
lgtm

**Review by AlexisRoe — APPROVED** [review]
lgtm

File: src/app/App.tsx (hunk 1 above)
Comment (monavorschulze) [reply src/app/App.tsx]: @AlexisRoe You're also right. The other div container is still there. I mistakenly inserted a single div tag.

**Review by lmachens — APPROVED** [review]
gj

**Pull request merged**